restli-tools/src/main/java/com/linkedin/restli/tools/idlcheck/CompatibilityInfo.java

```diff
@@ -28,7 +28,13 @@ public class CompatibilityInfo
   public enum Level
   {
     INCOMPATIBLE,
-    COMPATIBLE
+    COMPATIBLE,
+
+    /**
+     * Serialization and deserialization is backwards compatible, but the change may be mishandled by clients
```

Comment:
by reading this comment "Serialization and deserialization is backwards compatible" I still don't know what WIRE_COMPATIBLE means, I think you meant pegasus serializer's serialized result can be deserialized by old pegasus deserializer

Reply:
Yes, but also by new pegasus deserializers (even though this was never a problem).
The combinations: new writer, old reader, and new reader, old writer both work with new enum values. I will clarify the comment

Reply:
@junchuanwang is WIRE_COMPATIBLE a good name or do you have a different suggestion?

Reply:
`WIRE_COMPATIBLE` seems OK to me. It means that the change is compatible "over the wire" but not necessarily understood by the client.